### DISCUSSION TITLE
Nested sorting reliably breaks

### DISCUSSION THREAD

**Discussion by assertchris** [alpinejs/alpine#4433] [7. Bugs (archived)]
I'm experimenting with nested sorting, and it appears to break pretty reliably: https://codepen.io/assertchris/pen/OJKrzqB?editors=1000

Try to drag one of the 2 items into the nesting container that appears as you drag. When you drag it out, you get a nextSibling JS error (which I suspect is because the list is now empty, but I haven't been able to confirm this).

**Comment by ekwoka** [comment]
Likely fixed in #4361

**Comment by imade-nl** [comment]
I ran into the same problem. Latest version.
